force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +114,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
+        Integer vowels = 0;
+        str = str.toLowerCase();
+        
+        for(Integer i = 0; i < str.length(); i++) {
+        String ch = str.substring(i, i+1);
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
+        if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+            vowels++;
```

Comment:
Great implementation. For formatting, be sure to add a space before the `(` in your for loops and conditionals